Core/Systems/ThemeSystem/ThemeHandler.cs

```diff
@@ -92,6 +93,12 @@ public static ThemeIconProvider GetIconProvider<T>() where T : ThemeIconProvider
 			return allIconProvidersByType[typeof(T)];
 		}
 
+		/// <summary>Add an icon at runtime so every icon provider can draw it.</summary>
+		public static void RegisterAPIIcon(string key, Asset<Texture2D> tex)

```

Comment:
Does this mean its impossible for an API user to register multiple variants for the different styles? Ideally they would be able to register to each provider seperately and also have a fallback available